examples/web/src/index.tsx

```diff
@@ -2,17 +2,32 @@ import React from 'react';
 import { render } from 'react-dom';
 
 import { ApolloProvider, ApolloClient } from '@apollo/client';
-import { Query, Mutation, Subscription } from '@apollo/client/react/components';
+import { Query } from '@apollo/client/react/components';
 import gql from 'graphql-tag';
 import { InMemoryCache } from '@apollo/client/core';
 import { CachePersistor } from 'apollo3-cache-persist';
+import { PersistentStorage } from "apollo3-cache-persist/types";
 
 let persistor;
 
+class LocalStoragePersistedStorage implements PersistentStorage<string> {
```

Comment:
This is why we needed to go to typescript. Looks like types are not aligned - Still. 

Hoping to get that over the line in separate PR.

Reply:
Actually, if we have the wrapper in the package that would be cool and work outside the box.

Reply:
sure I can move it. Give me till 17:00 UTC+2, lunch-break is over